veejay-client build fail

Hi,

tried to build reloaded and it didn't work : 

gcc -DHAVE_CONFIG_H -I. -I..  -I.. -I../src -I/usr/local/include  -I/usr/include/gtk-3.0 -I/usr/include/pango-1.0 -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/sysprof-6 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/fribidi -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/x86_64-linux-gnu -I/usr/include/webp -I/usr/include/gio-unix-2.0 -I/usr/include/atk-1.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/x86_64-linux-gnu/dbus-1.0/include -pthread  -I/usr/include/x86_64-linux-gnu  -DGNOMELOCALEDIR=\""/usr/local/share/locale"\" -DRELOADED_DATADIR=\""/usr/local/share/reloaded"\"  -O2 -Wall -Wunused    -m64 -fPIC -DPIC -DFILE_OFFSET_BITS=64 -D_LARGE_FILES -msse -mfpmath=sse -fno-stack-protector -Wno-deprecated-declarations -MT multitrack.o -MD -MP -MF .deps/multitrack.Tpo -c -o multitrack.o multitrack.c
multitrack.c:371:78: error: initialization of ‘void (*)(void)’ from incompatible pointer type ‘void (*)(GtkWidget *, void *)’ {aka ‘void (*)(struct _GtkWidget *, void *)’} [-Wincompatible-pointer-types]
  371 |         { "button_gotostart", VIMS_VIDEO_GOTO_START, "button_gotostart.png", seq_gotostart },
      |                                                                              ^~~~~~~~~~~~~

[...]

multitrack.c:380:73: error: initialization of ‘void (*)(void)’ from incompatible pointer type ‘void (*)(GtkWidget *, void *)’ {aka ‘void (*)(struct _GtkWidget *, void *)’} [-Wincompatible-pointer-types]
  380 |         { "button_nextframe", VIMS_VIDEO_SKIP_FRAME, "button_skip.png", seq_nextframe },
      |                                                                         ^~~~~~~~~~~~~
multitrack.c:380:73: note: (near initialization for ‘button_template_t[8].f’)
multitrack.c:301:17: note: ‘seq_nextframe’ declared here
  301 | static  void    seq_nextframe(GtkWidget *w, gpointer data)
      |                 ^~~~~~~~~~~~~
make[2]: *** [Makefile:501: multitrack.o] Error 1
make[2] : on quitte le répertoire « /home/guilux/compilation/veejay/veejay-current/veejay-client/src »
make[1]: *** [Makefile:596: all-recursive] Error 1
make[1] : on quitte le répertoire « /home/guilux/compilation/veejay/veejay-current/veejay-client »
make: *** [Makefile:526: all] Error 2


Thanks !

--- a/veejay-current/veejay-client/src/multitrack.c
+++ b/veejay-current/veejay-client/src/multitrack.c
@@ -365,7 +365,7 @@ static struct
 	const char *name;
 	int vims_id;
 	const char *file;
-	void (*f)();
+	void (*f)(GtkWidget *, gpointer);
 } button_template_t[] =
 {
 	{ "button_gotostart", VIMS_VIDEO_GOTO_START, "button_gotostart.png", seq_gotostart },

